data.md

```diff
@@ -238,16 +238,23 @@ This stack also serves as a cons-list, so we provide some standard cons-list man
 -   `WS [ N .. W ]` access the range of `WS` beginning with `N` of width `W`.
 
 ```{.k .uiuck .rvk}
-    syntax WordStack ::= WordStack "++" WordStack [function]
- // --------------------------------------------------------
+    syntax WordStack ::= WordStack "++" WordStack [function, left]
```

Comment:
Making this right-associative would be more convenient for verification. Is there any reason to prefer left?